doc/ref_model/chapters/chapter07.md

```diff
@@ -43,7 +43,9 @@
   * [7.10.1 Networking Security Zoning.](#7.4.1)
   * [7.10.2 Encryption.](#7.4.2)
   * [7.10.3 Platform Patching.](#7.4.3)
-  * [7.10.4 Boot Integrity Measurement (TPM).](#7.4.4)
+  * [7.10.4 Boot Integrity Measurement (TPM).](#7.10.4)
+  * [7.10.5 Runtime Integrity Measurement (TPM).](#7.10.5)
+  * [7.10.6 NFVI & VIM.](#7.10.6)
```

Comment:
Remove period from headings. Please delete it from the existing ones in this chapter. I will also request to fix the references of § 7.8-§ 7.10 in the headings.

Reply:
Fixed in the last commit.